docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3.3"
+services:
+  sonar:
+    build:
+      dockerfile: Dockerfile
+      context: .
+    container_name: sonar_ecocode_ios
+    ports:
+      - "9000:9000"
+    networks:
+      - sonarnet
+    depends_on:
+      - db
+    environment:
+      SONAR_JDBC_USERNAME: sonar
+      SONAR_JDBC_PASSWORD: sonar
+      SONAR_JDBC_URL: jdbc:postgresql://db:5432/sonarqube
+      SONAR_ES_BOOTSTRAP_CHECKS_DISABLE: 'true'
+    volumes:
+      - type: bind
+        source: ./sonar-ios-plugin/target/ecocode-sonar-ios-plugin-1.0.2-SNAPSHOT.jar
```

Comment:
change version to `1.1.0-SNAPSHOT` beacause of important migration but with retro-compatibility